src/Components/Pages/TransferPage.tsx

```diff
@@ -101,7 +185,7 @@ const useStyles = makeStyles(({ constants, palette }: ITheme) =>
       borderRadius: 2,
       color: palette.additional["gray"][9],
       marginTop: constants.generalUnit,
-      fontSize: 11,
+      fontSize: 12,
```

Comment:
Confirm this size work with long addresses.

Reply:
Confirmed font size 12 work with long addresses.